app/controllers/shoes_controller.rb

```diff
@@ -48,6 +48,7 @@ def update
   def destroy
     @shoe = Shoe.find(params[:id])
     if @shoe.user == current_user
+      @shoe.reviews.destroy_all
```

Comment:
Nice attention to detail here, I would recommend instead to take advantage of ActiveRecord and adding `has_many :reviews, dependent: :destroy` to your Shoe model.